cmd/kubectl-gadget/traceloop.go

```diff
@@ -213,101 +117,23 @@ func runTraceloopStop(cmd *cobra.Command, args []string) error {
 }
 
 func runTraceloopList(cmd *cobra.Command, args []string) error {
-	client, err := k8sutil.NewClientsetFromConfigFlags(utils.KubernetesConfigFlags)
-	if err != nil {
-		return commonutils.WrapInErrSetupK8sClient(err)
-	}
-
-	tracesPerNode, err := getTracesListPerNode(client)
+	traces, err := utils.ListTracesByGadgetName("traceloop")
 	if err != nil {
-		return fmt.Errorf("failed to get the running traces: %w", err)
+		return err
 	}
 
-	var traces []tracemeta.TraceMeta
-	for _, tm := range tracesPerNode {
-		traces = append(traces, tm...)
-	}
-	sort.SliceStable(traces, func(i, j int) bool {
-		if traces[i].Namespace != traces[j].Namespace {
-			return traces[i].Namespace < traces[j].Namespace
-		}
-		if traces[i].Podname != traces[j].Podname {
-			return traces[i].Podname < traces[j].Podname
-		}
-		if traces[i].Containeridx != traces[j].Containeridx {
-			return traces[i].Containeridx < traces[j].Containeridx
-		}
-		if traces[i].TimeCreation != traces[j].TimeCreation {
-			return traces[i].TimeCreation < traces[j].TimeCreation
-		}
-		if traces[i].TimeDeletion != traces[j].TimeDeletion {
-			return traces[i].TimeDeletion < traces[j].TimeDeletion
-		}
-		return false
-	})
-
-	namespace, _ := utils.GetNamespace()
-
 	w := tabwriter.NewWriter(os.Stdout, 0, 0, 4, ' ', 0)
-	if !optionListNoHeaders {
-		if optionListFull {
-			fmt.Fprintln(w, "NODE\tNAMESPACE\tPOD\tPODUID\tINDEX\tTRACEID\tCONTAINERID\tSTATUS\tCAPABILITIES\t")
-		} else {
-			if !optionListAllNamespaces {
-				fmt.Fprintln(w, "POD\tPODUID\tINDEX\tTRACEID\tCONTAINERID\tSTATUS\t")
-			} else {
-				fmt.Fprintln(w, "NAMESPACE\tPOD\tPODUID\tINDEX\tTRACEID\tCONTAINERID\tSTATUS\t")
-			}
-		}
-	}
 
+	fmt.Fprintln(w, "TRACEID\tNAMESPACE\tPOD\tCONTAINER\tCONTAINERID")
 	for _, trace := range traces {
-		if trace.Containeridx == -1 {
-			// The pause container
-			continue
-		}
-
-		if trace.Namespace != namespace && !optionListAllNamespaces {
-			continue
+		var infos []traceloopTypes.TraceloopInfo
+		err := json.Unmarshal([]byte(trace.Status.Output), &infos)
+		if err != nil {
+			return err
```

Comment:
I think that if `trace.Status.Output == ""`, we should silently ignore this error and `continue` on the for loop. Otherwise, the `list` command and the `show` command cannot be run at the same time in different terminals.